api/src/LinkyLink/Startup.cs

```diff
@@ -61,8 +61,8 @@ public void ConfigureServices(IServiceCollection services)
                     options.JsonSerializerOptions.IgnoreNullValues = true;
                 });
 
-            // The following line enables Application Insights telemetry collection.
-            services.AddApplicationInsightsTelemetry(Configuration.GetSection("ApplicationInsights")["InstrumentationKey"]);
+            // Uncomment the following line to enable Application Insights telemetry collection.
+            //services.AddApplicationInsightsTelemetry(Configuration.GetSection("ApplicationInsights")["InstrumentationKey"]);
```

Comment:
I thought it was agreed upon to keep this line enabled?
https://github.com/Azure-Samples/openhack-production/pull/43#discussion_r375339060

Reply:
Enabling it. It must have been commented out part of merging it with master. Glad you noticed it.